tests/rest/admin/test_room.py

```diff
@@ -941,6 +941,52 @@ def _search_test(
         _search_test(None, "bar")
         _search_test(None, "", expected_http_code=400)
 
+    def test_search_term_non_ASCII(self):
```

Comment:
We usually use all lower-case for method names.

```suggestion
    def test_search_term_non_ascii(self):
```